org.freeorion.FreeOrion.json

```diff
@@ -0,0 +1,60 @@
+{
+  "id": "org.freeorion.FreeOrion",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "freeorion",
+  "rename-icon": "freeorion",
+  "rename-desktop-file": "freeorion.desktop",
+  "rename-appdata-file": "freeorion.appdata.xml",
+  "finish-args": [
+    "--socket=x11",
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--share=network"
+  ],
+  "modules": [
+    "shared-modules/glu/glu-9.0.0.json",
+    "shared-modules/glew/glew.json",
+    {
+      "name": "boost",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2",
+          "sha256": "7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7"
+        }
+      ],
+      "build-commands": [
+        "./bootstrap.sh --prefix=${FLATPAK_DEST}",
```

Comment:
You should build only the needed modules from boost.

Reply:
It needs all modules.